.github/workflows/contracts-e2e-tests-and-deploy.yaml

```diff
@@ -136,7 +137,7 @@ jobs:
 
   build_and_deploy_contracts:
     name: Deploy contracts on feature environment
-    if: ${{ (github.event_name == 'push') && (github.ref == 'refs/heads/benjamin') }}
+    if: (github.event_name == 'push' && github.ref_name == 'add-labels-for-deploying-contracts') || (github.event_name == 'pull_request' && github.event.action == 'labeled' && github.event.label.name == '[AZERO] DEPLOY-CONTRACTS')
```

Comment:
leftover from testing?

Reply:
this is now changed back to 'benjamin', had to test something

Reply:
@mikogs it is still not changed :)